SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -2,13 +2,13 @@ This document is a declaration of software quality for the `spdlog` external dep
 
 # `spdlog` Quality Declaration
 
-This quality declaration claims that `spdlog` is in the **Quality Level 4** category.
+This quality declaration claims that `spdlog` is in the **Quality Level 1** category.
 
-Though `spdlog` meets many of the criteria stated for software quality, there are key points preventing it from achieving a higher level category.
+Though `spdlog` meets many of the criteria stated for software quality.
 As `spdlog` itself is not a ROS package, many of these gaps could be reconciled in the vendor package that is used to control `spdlog` distribution within ROS.
 Engagement with the upstream development team to define policies for versioning, API/ABI stability and enforcing peer review would lead to higher quality, and could increase the level category for `spdlog` in a subsequent review.
```

Comment:
If we are justifying this as QL 1, we should just get rid of this whole paragraph.

Reply:
https://github.com/ros2/spdlog_vendor/pull/16/commits/f26283584e93662e173724ca439a28ca380fae40